### ISSUE TITLE
Reference strip in fulscreen not showing all thumbs

### ISSUE THREAD

**Issue by bsait** [openseadragon/openseadragon#1826]
Hi,
I have a sequence of images with a reference strip that works fine in a small div.
When I open full screen, the reference strip shows only the first few thumbs, which were loaded in the initial view. I can click on the empty placeholders in the strip, and the images show, but not the thumbs.
https://digital.bsa.ac.uk/results.php?keywords=epen&irn=145359

Any ideas?
Thanks

**Comment by iangilman** [comment]
Good catch! It looks like the reference strip responds to updates by calling `loadPanels`: 

https://github.com/openseadragon/openseadragon/blob/48a18a6a7d93b49c36bbb6ce6569876c1d9f62ef/src/referencestrip.js#L410

...But it never detects that it needs to update on 'full-screen', so it never calls `loadPanels`. It exhibits the same problem when you resize the window.

I think a reasonable fix would be for the reference strip to keep track of its viewer's size and inside its update function: 

https://github.com/openseadragon/openseadragon/blob/48a18a6a7d93b49c36bbb6ce6569876c1d9f62ef/src/referencestrip.js#L287

...it could check to see if the size changed, and call `loadPanels` if it has.

> ...and found another issue: the new ReferenceStrip does not respect the initially set referenceStripHeight. It does, however, respect the referenceStripSizeRatio.

It only uses `referenceStripHeight` if `referenceStripWidth` is also present. Seems like that makes it less than useful, since it would make sense to specify one without the other (so you have a specific height but an unlimited width, for instance). At the very least, the documentation for those parameters should be improved! See https://github.com/openseadragon/openseadragon/issues/1825 for related discussion. 

Would you be up for tackling either of these issues? If not, hopefully someone will, and it sounds like you have a work around at least. Thank you for reporting them!

**Comment by msalsbery** [comment]
This should be fixed as of #1872 

@bsait  The next version hasn't been released yet, but if you're still having an issue here can you test off a master branch build? Also there's [preview builds here](https://github.com/openseadragon/openseadragon/pull/1872#issuecomment-673608460)
